src/status.js

```diff
@@ -0,0 +1,28 @@
+export default class Status {
+  validation() {
+    if (this.checked === true) {
+      this.nextSibling.style['text-decoration'] = 'line-through';
+      this.nextSibling.style.color = '#909090';
+      // Save in Local Storage
+      Status.saveChanges();
```

Comment:
Use conditional shorthands. `(this.checked)`, `(this.checked === true)` is not necessary.

Reply:
Sure. Thanks. 💯